lib/vm.js

```diff
@@ -294,6 +299,8 @@ module.exports = {
   runInNewContext,
   runInThisContext,
   isContext,
+  getWrappedFunction,
+  getWrappedFunctionGeneric,
```

Comment:
I think we only want to have one export of this kind? Maybe this is already implied in the `WIP` tag here, just want to make sure it doesn’t get lost :)

Reply:
wanted to expose the generic version in case someone needs it. The `getWrappedFunction` is a smaller variant with only 2 args and default values.

Reply:
That said, an options object with defaults sounds better, so I'll probably do that instead.